### ISSUE TITLE
[new site support] Trinity College of Dublin

### ISSUE THREAD

**Issue by jaaschutz** [lovasoa/dezoomify#322]
### Site name and desciption
digitalcollections.tcd.ie

This is the digital collection of Trinity College in Dublin....an INCREDIBLE European resource with an amazingly huge collection. They even have the Famed Book of Kells on here. 

### Example URLs
 - https://digitalcollections.tcd.ie/home/index.php?DRIS_ID=MS58_003v
 - https://digitalcollections.tcd.ie/content/14/pages/MS58_001r/image_files

### Current error message
I confirmed that the site is using Seadragon. When I select Seadragon I get this error: 
*Error: Invalid XML:
https://digitalcollections.tcd.ie/content/14/pages/MS58_001r/image_files/10/1_0.jpg
(https://ophir.alwaysdata.net/dezoomify/zoommanager.js:191)*

When I use automatic I get 
*Error: Unable to find a proper dezoomer for:
https://digitalcollections.tcd.ie/content/14/pages/MS58_001r/image_files/10/1_0.jpg
(https://ophir.alwaysdata.net/dezoomify/dezoomers/automatic.js:30)*

**Comment by lovasoa** [comment]
Hello !
  Have a look at [the project's FAQ](https://github.com/lovasoa/dezoomify/wiki/Dezoomify-FAQ), where you'll find an explanation of how to locate the metadata file URL to pass to dezoomify. If you find the explanations hard to follow or would like to change something, don't hesitate. It's a wiki.

In the case of your example, it is

```
https://digitalcollections.tcd.ie/content/14/pages/MS58_001r/image.dzi
```

https://ophir.alwaysdata.net/dezoomify/dezoomify.html#https://digitalcollections.tcd.ie/content/14/pages/MS58_001r/image.dzi 


![canvas](https://user-images.githubusercontent.com/552629/64388628-f72dc600-d03f-11e9-9ccf-37cfa7fa3410.jpg)

**Comment by lovasoa** [comment]
I added support for dezooming directly from tile URLs for deepzoom images in d2dbd4f3cb255ce2056adabd49ca32cacace7c59
